app/src/modules/settings/routes/webhooks/collection.vue

```diff
@@ -170,6 +164,13 @@ function clearFilters() {
 </template>
 
 <style lang="scss" scoped>
+.deprecation-notice-wrapper {
+	padding: var(--content-padding);
```

Comment:
Specifically added it, otherwise it touches the table which doesnt look good imo, see:

![image](https://github.com/directus/directus/assets/14810858/6f67f527-0319-4d04-b10b-18b496bde6d9)

versus:

![image](https://github.com/directus/directus/assets/14810858/ad21b12d-94d8-4cf6-9d5f-ec5be64281a2)

Reply:
Sorry, meant no vertical padding on top (notice the large gap between header bar and deprecation notice)

Reply:
Ahhh thank you! Done.